parcels/examples/example_nemo_curvilinear.py

```diff
@@ -68,6 +69,29 @@ def test_nemo_curvilinear(mode):
     run_nemo_curvilinear(mode, outfile)
 
 
+def test_nemo_3D_samegrid():
+    data_path = path.join(path.dirname(__file__), 'NemoNorthSeaORCA025-N006_data/')
```

Comment:
So, you're adding new data in oceanparcels_website?

Reply:
No, this is a new unit test that uses the 3D nemo data that was there already for the 3d nemo tutorial. The point is that we want to test this on multiple filenames

Reply:
Oh yes sure. I was thinking there are only two time steps, so no defer loading, but it is not relevant in this test. my bad